benchmarks/experiment_runner.py

```diff
@@ -429,187 +489,170 @@ def parse_log_level(level: str):
 
   parser.add_argument(
       "--log-level",
-      default="info",
+      default=logging.INFO,
       choices=[
           logging.CRITICAL,
           logging.ERROR,
           logging.WARNING,
           logging.INFO,
           logging.DEBUG,
       ],
-      type=parse_log_level,
+      type=_parse_log_level,
       help="Specify log level.",
   )
```

Comment:
Maybe it's not an issue for this PR, but I'd think we want string choices instead of integers. A minimal program with only this arguments gives:

```python
usage: log.py [-h] [-l {50,40,30,20,10}]

options:
  -h, --help           show this help message and exit
  -l {50,40,30,20,10}  Specify log level.
```

Reply:
Agreed, that does not look nice. Will address that in a follow-up

Reply:
Addressed in https://github.com/pytorch/xla/pull/6188